book_source/developers_guide/Coding_style.Rmd

```diff
@@ -0,0 +1,83 @@
+Coding Style
+------------
```

Comment:
We should use consistent header styles, especially in the section on code styles.